src/frontend/src/sass/_form.scss

```diff
@@ -0,0 +1,3 @@
+.form-heading {
+  font-weight: 700;
```

Comment:
This should be part of the text tags (h1, h2, h3, etc). We should not be defining font-size like this.

Reply:
I discuss this with @Bottle7. Hi @Bottle7 do you have thoughts?